docs/administration/security/octopus-tentacle-communication/index.md

```diff
@@ -50,11 +50,17 @@ Tentacle plays the role of server and Octopus as the client:
 
 Octopus plays the role of server and Tentacle as the client:
 
-1. The Tentacle establishes the HTTPS connection with Octopus.
+1. The Tentacle establishes the TLS connection with Octopus.
 2. Octopus presents its certificate as the server certificate allowing the Tentacle to verify the identity of Octopus.
 3. The Tentacle presents its certificate as a client certificate so Octopus can verify the identity of the Tentacle.
 4. Once the identity of the Octopus and Tentacle have been established the connection is held open and Octopus will start issuing commands to the Tentacle.
 
+### Data transport
+
+Octopus uses [Halibut](https://github.com/OctopusDeploy/Halibut) to communicate. This is based on raw TCP, not HTTP - it is not possible to add HTTP headers to this communication channel.
```

Comment:
[Nitpicking] I'd personally drop the word 'raw' as I don't think it really helps to clarify particularly for ESL readers.

```suggestion
Octopus uses [Halibut](https://github.com/OctopusDeploy/Halibut) to communicate. This is based on TCP, not HTTP - it is not possible to add HTTP headers to this communication channel.
```